src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -744,7 +744,7 @@ class SerializeClosure;
   template(toFileURL_signature,                             "(Ljava/lang/String;)Ljava/net/URL;")                 \
   template(url_array_classloader_void_signature,            "([Ljava/net/URL;Ljava/lang/ClassLoader;)V")          \
                                                                                                                   \
-  template(jdk_crac_Core,                          "jdk/crac/Core")                                               \
+  template(jdk_crac_Core,                          "jdk/internal/crac/mirror/Core")                               \
```

Comment:
I think I've missed this one in the first review - shouldn't the first parameter be fixed as well?